### ISSUE TITLE
Failed to add some URI(s)

### ISSUE THREAD

**Issue by krzystke** [jaedb/Iris#107]
Let me copy beginning from other Issue: I'm able to connect to Spotify via my premium account (account created with Facebook) with my content visible in the UI; Playlists, Daily Recommend, etc. However, trying to play any content (Artist, Radio, Playlist, or even manually adding URL's from Spotifys Desktop App) results in the same error message "failed to add some url(s)"

I cannot generate app password in spotify account page, but I generated app password via facebook security page. I don't know what I need to write in config so I am trying spotify client number, e-mail used with facebook, login@facebook.com - nothing works. I can login with e-mail credentials at spotify windows app but Iris (or Mopify) still doesn't work.

**Comment by Informatic** [comment]
It won't work. This is not an issue with Iris, but your Mopidy-Spotify integration. Official solution for screwed up "get device password" thing on Facebook Login accounts is to contact the support, or just create a new account. Did that 5 minutes ago.

**Comment by krzystke** [comment]
I don't know is there a problem with Iris in general or with my installation, I tried logging in with 'normal' spotify account which works great on other rpi with spotmob (older Iris) but still same error about uri(s). Local files are working good.

**Comment by jaedb** [comment]
@krzystke have you tried the official solution? Try the suggestions mentioned on the [Mopidy-Spotify](https://github.com/mopidy/mopidy-spotify#dependencies) readme.

**Comment by krzystke** [comment]
I tried official solution but I got still same error. I tried using mopidy again after reinstall and everything worked, I just got error "No such user" at the start, but that's all, music was playing. Failed to add some URI(s) problem disappeared. Finally I just moved facebook created account to Spotify standalone, and now everything is working fine without errors.

**Comment by jaedb** [comment]
@krzystke Glad to hear you're up and running. It'd be great to see a more simple install tool (and support for Windows) for less computer-savvy users to get onto Mopidy - all things in time though!

**Comment by krzystke** [comment]
@jaedb I created tutorial which maybe is not automatic like install tools, but it explains quite good what is going on, and user just need to copy and paste command after command to terminal. No additional knowledge needed. I also created one long command (comand1 && command2 && command3...) which also works. It installs Mopidy + Alsa drivers and utils + Spotify + Iris + Alarm Clock. I don't know if someone anywhere really wants it, but if You think I can maybe publish somewhere or you want it to look at it and maybe it will help, I am glad to publish it (just after I improve there few things and translate to English from Polish).
